HGCalAnalysis/plugins/HGCalAnalysis.cc

```diff
@@ -386,6 +424,27 @@ HGCalAnalysis::HGCalAnalysis(const edm::ParameterSet& iConfig) :
 	t->Branch("multiclus_slopeY", &multiclus_slopeY);
 	t->Branch("multiclus_cluster2d", &multiclus_cluster2d);
 	t->Branch("multiclus_cl2dSeed", &multiclus_cl2dSeed);
+	t->Branch("multiclus_pcaAxisX", &multiclus_pcaAxisX);
+	t->Branch("multiclus_pcaAxisY", &multiclus_pcaAxisY);
+	t->Branch("multiclus_pcaAxisZ", &multiclus_pcaAxisZ);
+	t->Branch("multiclus_pcaPosX", &multiclus_pcaPosX);
+	t->Branch("multiclus_pcaPosY", &multiclus_pcaPosY);
+	t->Branch("multiclus_pcaPosZ", &multiclus_pcaPosZ);
+	t->Branch("multiclus_eigenVal1", &multiclus_eigenVal1);
+	t->Branch("multiclus_eigenVal2", &multiclus_eigenVal2);
+	t->Branch("multiclus_eigenVal3", &multiclus_eigenVal3);
+	t->Branch("multiclus_eigenSig1", &multiclus_eigenSig1);
+	t->Branch("multiclus_eigenSig2", &multiclus_eigenSig2);
+	t->Branch("multiclus_eigenSig3", &multiclus_eigenSig3);
+	t->Branch("multiclus_siguu", &multiclus_siguu);
+	t->Branch("multiclus_sigvv", &multiclus_sigvv);
+	// t->Branch("multiclus_sigrad", &multiclus_sigrad);
+	// t->Branch("multiclus_radsiguu", &multiclus_radsiguu);
+	// t->Branch("multiclus_radsigvv", &multiclus_radsigvv);
```

Comment:
Do you intend to use these variables? If not, would be better to remove them completely (also below)